common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java

```diff
@@ -85,6 +94,45 @@
         return result;
     }
 
+    public static boolean renewFociTokens(final String clientId,
+                                          final String redirectUri,
+                                          final ICacheRecord cacheRecord)
+            throws ClientException, IOException {
+        final MicrosoftStsOAuth2Configuration config = new MicrosoftStsOAuth2Configuration();
+
+        //Get authority url
+        final Uri.Builder requestUrlBuilder = new Uri.Builder();
+        final String tenantId = cacheRecord.getAccount().getRealm();
+        requestUrlBuilder.scheme(REDIRECT_SSL_PREFIX)
```

Comment:
Let's break this line into a few different statements for readability

Reply:
Addressed.